### ISSUE TITLE
evaluation.m

### ISSUE THREAD

**Issue by qhu009** [earthat/VANET-Simulation-in-MATLAB#3]
Hi Earthat,
I encounter an issue when I run evaluation.m. Firstly,  when I input the number of the node and the number of the RSU, I got the following error. When I check the code, I notice the code didn't go through the first loop. 

>> evaluation(10,3)
Undefined function or variable 'distnc'.

Error in evaluation (line 7)
total_dist=sum(distnc); % total distnace from source to destination

May I ask for some help?

Thank you very much.

Best regards,

Steven

**Comment by earthat** [comment]
kindly check `numel(nodtble)`. probably this value is either 1 or zero

**Comment by qhu009** [comment]
Yes, Earthat. the value is 1. But I have a question. The whole purpose is to configure 1 RSU and multiple nodes. Then I want to see pcktloss,throughput as the number of the nodes increasing. 
For the existing code, I try to call function 
function [E,pcktlossrate,total_dist,pcktloss,thrgput]=evaluation(nodtble,node_rsu)

I put the following line in the console. 
evaluation(10,3)

my understanding is nodtble is 10 and node_rsu is 3. Am I right?

**Comment by qhu009** [comment]
Hi Earthat, Steven again, I rechecked the code, I notice the UrbanCitySimu.m will trigger the evaluation function. I also notice at the end of UrbanCitySimu.m, you have the code to generate plot figures. May I ask where to find those figures. I am so sorry, I am very new for MATLAB. Sorry for the inconvenience
